src/sentry/celery.py

```diff
@@ -83,6 +84,10 @@ def good_use_of_pickle_or_bad_use_of_pickle(task, args, kwargs):
 class SentryTask(Task):
     Request = "sentry.celery:SentryRequest"
 
+    def delay(self, *args, **kwargs):
+        kwargs["__start_time"] = datetime.now().timestamp() * 1000
+        return super().delay(*args, **kwargs)
+
     def apply_async(self, *args, **kwargs):
```

Comment:
Probably should do this in `apply_async` as well

Reply:
@wedamija Actually, are we already recording the timing with `with metrics.timer("jobs.delay", instance=self.name):`?

Reply:
No, that's just timing how long it takes to send the task to celery afaict

Reply:
@wedamija hmmm I'm not sure how to get this to work for `apply_async` since we call `Task.apply_async` directly.